os/board/rtl8721csm/src/component/common/mbed/hal/serial_api.h

```diff
@@ -143,6 +143,16 @@ void serial_irq_handler(serial_t *obj, uart_irq_handler handler, uint32_t id);
   */
 void serial_irq_set(serial_t *obj, SerialIrq irq, uint32_t enable);
 
+/**
+  * @brief  Enable/Disable UART loopback mode for testing.
+  * @param  obj: uart object define in application software.
+  * @param  enable/disable the loopback mode
+  * 		@arg 0 disable
+  *		@arg 1 enable
+  * @retval none
+  */
+void serial_enable_loopback(serial_t *obj, bool enable);

```

Comment:
This function can disable loopback so that function name could be mismatched.
How about `serial_control_loopback`?

Reply:
corrected